modal/functions.py

```diff
@@ -1386,7 +1387,7 @@ async def get_current_stats(self) -> FunctionStats:
 Function = synchronize_api(_Function)
 
 
-class _FunctionCall(_Object, type_prefix="fc"):
+class _FunctionCall(_Object, type_prefix="fc"):  # type: ignore[call-arg]
```

Comment:
I'm less familiar with `type_prefix`, but on a grep looks like it adds a two char prefix to objects.

Reply:
@freider I've removed this. I was getting mypy error previously but might also have been when I was doing mypy checks before running inv etc.